### ISSUE TITLE
[FR]: maximum horizontal size of columns

### ISSUE THREAD

**Issue by BeastyBlacksmith** [ronisbr/PrettyTables.jl#126]
Currently I am working with the latex backend and if entries get too long they don't fit the page.
Thus it would be good to say, e.g. break lines if they exceed 3cm or similar.

I don't know how useful this would be for other backends.

**Comment by ronisbr** [comment]
Hi @BeastyBlacksmith 

Hum, it seems interesting! The problem is: I have no idea how to get the size of an entry.

One workaround should be changing the type of the column to `p` with a specific size. We need to think about a good API to do this. The implementation should be easy. Any ideas?

**Comment by BeastyBlacksmith** [comment]
I think we could extend `alignment` to accept things like `:p => "3cm"`

**Comment by ronisbr** [comment]
It would not be nice because it will change the type of `alignment`. It is a keyword that is passed to all backends as a `Vector{Symbol}`. In this case, we might have a performance problem due to type instabilities.

**Comment by BeastyBlacksmith** [comment]
an alternative would be to introduce a new keyword `column_widths` being a `Vector{String}`

**Comment by ronisbr** [comment]
Yes, it can be done! `column_widths` is not something common to all backends. What it should contain?

**Comment by BeastyBlacksmith** [comment]
So, I would map
```julia
pretty_table(df, backend = Val(:latex), alignment = :l, column_widths = "3cm") # \begin{tabular}{p{3cm}p{3cm}...}
pretty_table(df, backend = Val(:latex), alignment = :l, column_widths = ["4em", ""]) # \begin{tabular}{lp{4em}}
pretty_table(df, backend = Val(:latex), alignment = :r, column_widths = ["4em", ""]) # error, since p is left aligned
```
Does that make sense?

**Comment by ronisbr** [comment]
Yes, seems good! I will just change the name to `columns_width` to keep it equal as in text backend.

**Comment by ericphanson** [comment]
+1 for this!
